packages/multi-step/src/utils/crypto.ts

```diff
@@ -0,0 +1,31 @@
+import * as crypto from 'crypto';
+import { User } from '@accounts/types';
+
+export function createServiceId(user: User): string {
+  const hash = crypto.createHash('sha256');
+
+  hash.update(user.id);
+
+  return hash.digest('hex');
+}
+
+export function createToken(): { token: string; hash: string } {
```

Comment:
@ozsay Is that the token we send back to the client?

Reply:
yes